crates/wasmcloud-host/src/host.rs

```diff
@@ -95,7 +95,8 @@ impl HostBuilder {
 
     /// Enables remote control of the host via the lattice control protocol by providing an instance
     /// of a NATS connection through which control commands flow. Remote control of hosts is
-    /// disabled by default.
+    /// disabled by default. For security reasons, you should not use the same NATS connection or
+    /// security context for both RPC and lattice control.
```

Comment:
I'm interested in this, why not? What I can think of immediately is that for some reason if your connection to the RPC was compromised, you can shut it down without affecting the lattice. Is there more to the story there?

Reply:
That's basically it. Separate the connections and it minimizes the blast radius due to compromise. Someone could compromise _either_ the RPC connection _or_ the lattice control connection, but not both.